python/ml4ir/applications/ranking/tests/data/csv/synthetic/feature_config.yaml

```diff
@@ -0,0 +1,114 @@
+query_key:
```

Comment:
@calliebradley Is this the same as the feature_config under `applications/ranking/tests/data/tfrecord/feature_config.yaml`?

Reply:
@lastmansleeping Almost - I need/want to use query_str instead of query_key in order to test the building of synthetic query IDs. The random sampling would be a no-op using query_key since it's only one char long.

Reply:
If you're ok changing tfrecord/feature_config.yaml to use query_str, we could consolidate.

Reply:
It does have `query_str`, right?